mypyc/test-data/irbuild-tuple.test

```diff
@@ -162,3 +162,35 @@ L4:
     r8 = None
     return r8
 
+[case testNamedTupleAttribute]
+from typing import NamedTuple
+
+NT = NamedTuple('NT', [('x', int), ('y', int)])
```

Comment:
Might be worth adding a unit test for a NamedTuple that inherits from another, something like the example from #8663.